clients/tabby-agent/src/lsp/protocol.ts

```diff
@@ -177,6 +177,11 @@ export namespace ConfigurationRequest {
  * [Tabby] Defines the config supported to be changed on the client side (IDE).
  */
 export type ClientProvidedConfig = {
+  proxy?: {
+    authorization: string;
+    url: string;
+    noProxy: string | string[];
```

Comment:
```suggestion
```

I recommend removing this option for convenient of implementation in first step

Reply:
Updated.